### ISSUE TITLE
mta-sts-query documentation mentions verbosity, but it's not implemented

### ISSUE THREAD

**Issue by PAStheLoD** [Snawoot/postfix-mta-sts-resolver#62]
**Describe the bug**
mta-sts-query documentation mentions verbosity, but it's not implemented

[client's](https://github.com/Snawoot/postfix-mta-sts-resolver/blob/master/postfix_mta_sts_resolver/__main__.py#L9) ArgParse setup lacks -v, but [daemon's](https://github.com/Snawoot/postfix-mta-sts-resolver/blob/master/postfix_mta_sts_resolver/daemon.py#L27
) clearly has it.

and using docker to run the client, -h output shows that -v is not implemented.

```
# docker exec postfix-mta-sts-resolver mta-sts-query -h
usage: mta-sts-query [-h] domain [known_version]

positional arguments:
  domain         domain to fetch MTA-STS policy from
  known_version  latest known version (default: None)

optional arguments:
  -h, --help     show this help message and exit
```

Probably the docs should be fixed first, and later the client could be enhanced to support debug logging.

**Comment by Snawoot** [comment]
Fix released in version 0.8.2. New Docker images will be available in few minutes.

**Comment by PAStheLoD** [comment]
Oh wow, that was extremely fast :) Thanks for this great software!
